styles.css

```diff
@@ -347,32 +323,27 @@ footer {
   color: #898888;
 }
 
+/* About page styles */
 
+#about-section {
+  background-image: url("./img/Hero.jpg");
+  background-size: cover;
+  background-repeat: no-repeat;
+  text-align: center;
+}
```

Comment:
- [ ] [REQUIRED] Good job so far! Please give correct style to `main page` of the `About` section's `Text elements` like a [Behance Design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) 🙏🏼 

![image](https://github.com/Xghayor/Music-Festival/assets/63915024/bfdd8288-eead-4229-b2f2-fe5f9ba13a8c)